src/calendar/day.tsx

```diff
@@ -0,0 +1,90 @@
+import cx from 'classnames'
```

Comment:
1-1 transform from the previous file `src/calendar/day.jsx` no logical change has been applied